app/pulp/app/viewsets/repository.py

```diff
@@ -124,6 +124,16 @@ class PublisherViewSet(NamedModelViewSet):
     queryset = Publisher.objects.all()
     filter_class = PublisherFilter
 
+    def destroy(self, request, pk):
```

Comment:
I am not sure if it is worth a docstring here, since all the detailed routes will have the same set of arguments.
But this `pk` is a keyword argument and its name matters, it's a lookup field. In case of `RepositoryViewSet`, for example, it's not `pk` but `name`.
In [DRF docs](http://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing) they define it like `pk=None` just to show that it is a keyword argument, `pk` can't be `None`, it will be always specified.

To be more clear what `pk` is, is anyone in favor of `pk=None`, docstring or both?